frontend/src/features/rollout-announcement/components/NewFeatureTag.tsx

```diff
@@ -0,0 +1,20 @@
+import { IoRocketSharp } from 'react-icons/io5'
+import { Icon, Text } from '@chakra-ui/react'
+
+import Badge from '~components/Badge'
+
+export const NewFeatureTag = (): JSX.Element => {
+  return (
+    <Badge
+      variant="subtle"
```

Comment:
nit: we can just put color="secondary.500" on the parent Badge component and do not need extra declarations in the children components.